src/inc/CMakeLists.txt

```diff
@@ -30,13 +30,13 @@ get_include_directories(MIDL_INCLUDE_DIRECTORIES)
 
 
 # Run custom midl command over each idl file
-FIND_PROGRAM( MIDL midl.exe )
+#FIND_PROGRAM( MIDL midl.exe )
```

Comment:
We should not hardcode the tool. Why was it needed? The midl.exe doesn't seem to be platform specific. 
Also. the FIND_PROGRAM search path is configurable if you need to let it find the exe in a different place. For example by setting the CMAKE_PROGRAM_PATH.